lib/krill.js

```diff
@@ -624,8 +629,10 @@ function krillPrimPrintLDAP(pred) {
 	var key = sanityCheckResult.key;
 	var nbKeys = sanityCheckResult.nbKeys;
 
-	if (nbKeys === 0)
-		return ('(*)');
+	// Serializing an individual predicate with a number of keys !== 1 --
```

Comment:
This module uses only C-style comments so far.  Would you mind using them here, too?